### DISCUSSION TITLE
Adaptive grid

### DISCUSSION THREAD

**Discussion by HPL001** [idaholab/moose#29518] [Q&A Meshing]
The background is that I want to simulate the sintering of two particles. I have adopted the following mesh Settings, I hope to use adaptive mesh, but I do not know how to set, hope to get your help.
![image](https://github.com/user-attachments/assets/97f9912b-9e2a-4a26-bede-49f27b9aeba0)
 By the way, how is this uniform_refine defined, I don't seem to find an explanation of it in the help document.

**Comment by GiudGiud** [comment]
Hello

Uniform refinement is a step after mesh generation where each 2D element is split in 4, and each 3D element in 8.
There's not much to explain though you can look at the code that does it lin libmesh.

Adaptive meshing should be considered once you have a working model to improve performance. You dont want to add this complexity too early. 
https://mooseframework.inl.gov/syntax/Adaptivity/index.html

**Reply by HPL001** [reply]
Thank you for your help. I think I know what to do.
